README.md

```diff
@@ -52,3 +52,29 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+#Testing the App
+Unit Tests written for app. Tests to be run using command
+$ poetry run pytest
+
+# Running/building app via Docker use the following commands in Prod/dev environments
+# Note local .env file is passed in during Docker run.
+
+# running in Production env 
+docker build --target production --tag todo-app:prod .
+
+docker run --env-file ./.env -p 5001:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/opt/todo_app" todo-app:prod
```

Comment:
I know you said this was resolved on Slack, but note that the command here still has the mismatching ports!

Also, as this is our prod build, we don't necessarily want it to "hot reload" since we don't expect people to be making live changes, so we can get rid of the bind mount here
```suggestion
docker run --env-file ./.env -p 8000:8000 todo-app:prod
```